### DISCUSSION TITLE
uWS C API

### DISCUSSION THREAD

**Discussion by cirospaciari** [oven-sh/bun#1414] [Ideas]
@Jarred-Sumner 
The solution in https://github.com/oven-sh/bun/blob/0bfd00e734ebe6093fd69badbc1d733d039f3d6a/src/deps/libuwsockets.cpp
looks a lot with my PR https://github.com/uNetworking/uWebSockets/pull/1395 (https://github.com/cirospaciari/uWebSockets/blob/capi-complete-with-ssl-and-samples/capi/libuwebsockets.cpp)

I will update today and will update a lot (inclusive adding more features like Http3 Support soon), Can i Help with PRs to Bun adding these features? I will be very happy if i can help you with Bun.

Thanks!

**Comment by Jarred-Sumner** [comment]
That would be awesome 

Yes, I originally copy pasted the code from your PR for the C bindings - thank you for that. Made a small number of tweaks over time since that
